text/0007-gas-price-adjustment.md

```diff
@@ -0,0 +1,135 @@
+# Daily Gas Price Adjustment Algorithm
+
+## Summary
+
+[summary]: #summary
+
+CEP PR: [casperlabs/ceps#0004](https://github.com/casperlabs/ceps/pull/0004)
+
+Gas price is regulated algorithmically. A price floor exists and adjusts daily, based on fullnesses of blocks from the previous day, in order to absorb volatility and reduce congestion.
+
+## Motivation
+
+[motivation]: #motivation
+
+Gas prices are known to be subject to high volatility both in the short term and the long term. This is due to a combination of factors such as inelastic supply, [daily demand cycle](https://solmaz.io/2019/10/21/gas-price-fee-volatility/) and shifting trends.
+
```

Comment:
Please include 'CasperLabs sees these issues with volatility as a key barrier to the adoption of public blockchain infrastructure by businesses.  It is also recognized that price stability alone is not enough, scalability is also a core concern in this arena.'